changelog.md

```diff
@@ -1,5 +1,23 @@
 
 
+## 3.3.0 / 2020-12-xx
+
+### New Features
+
+- New activity feeds available per-space and per-user. You can now see what happened during your absence.
```

Comment:
```suggestion
- New personal and per-space activity feeds help you keep up with what is going on in Tracim: you will be able to notice new contents, new members and other updates in real time at a glance```

Reply:
I would remove the New as we are already in "New Features"

Reply:
Done